x-pack/plugins/canvas/public/components/workpad_page/workpad_static_page/static_workpad_page.js

```diff
@@ -23,9 +24,11 @@ export class StaticWorkpadPage extends PureComponent {
         data-shared-items-container
         style={{ ...pageStyle, ...animationStyle, height, width }}
       >
-        {elements.map(element => (
-          <ElementWrapper key={element.id} element={element} />
-        ))}
+        {elements
+          .filter(node => !isGroupId(node.id))
```

Comment:
nit: you could avoid iterating this array twice with a reduce, but it's unlikely to matter much...